idaes/models_extra/gas_solid_contactors/unit_models/fixed_bed_1D.py

```diff
@@ -0,0 +1,2047 @@
+            self.length_domain,
+            domain=Reals,
+            initialize=1.0,
+            doc="Particle Nusselt number",
+            units=pyunits.dimensionless,
+        )
+        self.gas_solid_htc = Var(
+            self.flowsheet().time,
+            self.length_domain,
+            domain=Reals,
+            initialize=1.0,
+            doc="Gas-solid heat transfer coefficient",
+            units=units_meta_gas("heat_transfer_coefficient"),
+        )
+
+        # Solid phase variables
+        self.solid_phase_area = Var(
+            self.flowsheet().time,
+            self.length_domain,
+            domain=Reals,
+            initialize=1,
+            doc="Solid phase area",
+            units=units_meta_solid("area"),
+        )
+
+        self.solid_material_holdup = Var(
+            self.flowsheet().time,
+            self.length_domain,
+            solid_phase.property_package.component_list,
+            domain=Reals,
+            initialize=1.0,
+            doc="Solid material holdup per unit length",
+            units=units_meta_solid("mass") / units_meta_solid("length"),
+        )
+
+        self.solid_material_accumulation = DerivativeVar(
+            self.solid_material_holdup,
+            wrt=self.flowsheet().config.time,
+            doc="Solid material accumulation",
+            units=units_meta_solid("mass")
+            / units_meta_solid("length")
+            / units_meta_solid("time"),
+        )
+
+        if self.config.energy_balance_type != EnergyBalanceType.none:
+            # Solid phase - gas to solid heat transfer
+            self.solid_phase_heat = Var(
+                self.flowsheet().time,
+                self.length_domain,
+                domain=Reals,
+                initialize=0.0,
+                doc="Heat transferred per unit length",
+                units=units_meta_solid("power") / units_meta_solid("length"),
+            )
+
+            self.solid_energy_holdup = Var(
+                self.flowsheet().time,
+                self.length_domain,
+                initialize=1,
+                doc="Solid phase energy holdup",
+                units=units_meta_solid("energy") / units_meta_solid("length"),
+            )
+
+            self.solid_energy_accumulation = DerivativeVar(
+                self.solid_energy_holdup,
+                initialize=0,
+                wrt=self.flowsheet().config.time,
+                doc="Solid energy accumulation",
+                units=units_meta_solid("power") / units_meta_solid("length"),
+            )
+
+        # =========================================================================
+        # Add performance equations
+
+        # ---------------------------------------------------------------------
+        # TODO - Deactivate/delete sum_component_eqns of initial conditions for
+        #  all space in gas_phase as currently no way to tell CV1D to not build
+        #  sum_eqns for initial conditions
```

Comment:
This is not correct - CV1D actually accounts for this automatically and will always set `defined_state=True` at the inlet point.